.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
```

Comment:
According to https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments, repos enabled in or after 2015 will default to the faster container architecture.